runtime/ftplugin/debchangelog.vim

```diff
@@ -203,17 +203,17 @@ function <SID>MakeMenu()
     menu Changelog.-sep-				<nul>
 
     amenu Changelog.Set\ &Distribution.&unstable	:call Distribution("unstable")<CR>
```

Comment:
@adaext Since `&Changelog` is used on line 200, should it be used for all the menu definitions and enable/disable commands?

Reply:
> @adaext Since `&Changelog` is used on line 200, should it be used for all the menu definitions and enable/disable commands?

Thanks, you are right. I've fixed it in the new commit.